src/silverlabnwb/nwb_file.py

```diff
@@ -128,9 +132,11 @@ def create_nwb_file(self, folder_path, session_id):
         :param session_id: the unique session ID for this experiment
         :returns: (speed_data, expt_start_time) for passing to import_labview_data
         """
+
```

Comment:
A style point: I personally prefer the internal function without the space around it, but not massively concerned :)